packages/rum-core/src/domain/configuration.ts

```diff
@@ -68,6 +70,7 @@ export interface RumConfiguration extends Configuration {
   version?: string
   subdomain?: string
   customerDataTelemetrySampleRate: number
+  traceContextInjection?: TraceContextInjection
```

Comment:
I think it can be undefined in the `RumInitconfiguration`, we just use the default value. 

But this is a good point. I made it always defined in RumConfiguration.